packages/client/hmi-client/src/components/project/tera-resource-sidebar.vue

```diff
@@ -109,7 +109,15 @@
 						force-fallback
 					>
 						<tera-asset-icon :asset-type="assetItem.pageType as AssetType" />
-						<span class="p-button-label">{{ assetItem.assetName }}</span>
+						<span class="p-button-label">
+							{{ assetItem.assetName }}
+							<tera-progress-spinner
+								v-if="inProgressAssetsIds[type] && inProgressAssetsIds[type].includes(assetItem.assetId)"
+								:font-size="1"
+								is-centered
+								style="float: right"
+							/>
+						</span>
```

Comment:
I uploaded a pdf, and I can see the spinner. But it seems to spin forever even though the extraction stuff seemed to have finished.